src/InteractiveWindow/Editor/InteractiveWindowResources.resx

```diff
@@ -219,4 +219,10 @@
   <data name="Usage" xml:space="preserve">
     <value>Usage:</value>
   </data>
+  <data name="WaitTitle" xml:space="preserve">
+    <value>Copy</value>
+  </data>
+  <data name="WaitMessage" xml:space="preserve">
+    <value>Formatting selection</value>
```

Comment:
When the wait dialog shows, it should explain to the user what will happen when they click ```Cancel```. In this case, it's not that the entire copy will be cancelled, so try to come up with something more specific here. My two-second example is "Generating Rich Text Format representation." or something.